gradle.properties

```diff
@@ -12,6 +12,6 @@ jedisVersion=2.4.1
 junitVersion=4.12
 restAssuredVersion=2.4.1
 toolsVersion=2.0.3-final
-vertxVersion=3.3.3
+vertxVersion=3.4.2

```

Comment:
With this change you not only change the vert.x version of the redis-client but also all other vert.x dependencies such as core, web and unit. Didn't you just wanted to update redis-client?